src/components/tools/text-tool.tsx

```diff
@@ -86,6 +87,14 @@ interface IState {
   value?: Value;
 }
 
+// create a global keylistener so that we know if the selection modifier keys are in use
+// when we get a change event
+let selectionModiferKeyDown = false;
+const globalOnKeyDown = (e: MouseEvent) => selectionModiferKeyDown = hasSelectionModifier(e);
+const globalOnKeyUp = (e: MouseEvent) => selectionModiferKeyDown = false;
```

Comment:
```suggestion
const globalOnKeyDown = (e: KeyEvent) => selectionModiferKeyDown = hasSelectionModifier(e);
const globalOnKeyUp = (e: KeyEvent) => selectionModiferKeyDown = hasSelectionModifier(e);
```
This will give an incorrect result in the case of two modifier key-downs followed by a modifier key-up. Seems like calling `hasSelectionModifier(e)` on keyUp should give the correct result.

Reply:
Good catch.  I updated the code to handle more than one option key being pressed at the same time: https://github.com/concord-consortium/collaborative-learning/pull/530/commits/1d201b5acd0cdc2e53ad9c516460605f18e55fec